packages/client-generator-ts/src/generateClient.ts

```diff
@@ -78,6 +78,7 @@ export interface BuildClientResult {
   prismaClientDmmf: DMMF.Document
 }
 
+// Note: this breaks
```

Comment:
Left over comment or what is its purpose? 🤔

Reply:
Removed